packages/components/src/components/action-bar/action-bar.tsx

```diff
@@ -108,6 +109,42 @@ export class ActionBar extends LitElement {
     this.expandToggleEl = el;
   };
 
+  private handleToolbarKeyDown = (event: KeyboardEvent): void => {
+    this.queryAndStoreActions();
+    const actions = this.actions.filter((action) => !action.disabled);
+    const current = document.activeElement as Action["el"];
```

Comment:
You might need to check the element's tag since the active element can be something other than an action (e.g., when you tab to the expand button then tab again, `current` will be the `action-bar`).